papas/pfalgo/pfblock.py

```diff
@@ -26,7 +26,7 @@ class PFBlock(object):
     temp_block_count = 0 #sequential numbering of blocks, not essential but helpful for debugging
 
     
-    def __init__(self, element_ids, edges, subtype): 
+    def __init__(self, element_ids, edges, index, subtype): 
```

Comment:
index should be added to the docstring (all new parameters should be added to the docstrings)